runtime/permissions.rs

```diff
@@ -174,12 +176,26 @@ pub struct WriteDescriptor(pub PathBuf);
 #[derive(Clone, Eq, PartialEq, Hash, Debug, Default, Deserialize)]
 pub struct NetDescriptor(pub String, pub Option<u16>);
 
+fn is_cidr(addr: &str) -> bool {
+  let res: Result<IpNet, _> = addr.parse();
+  if res.is_err() {
+    return false;
+  }
+  let res = res.unwrap();
+
+  res.prefix_len() < res.max_prefix_len()
+}
+
 impl NetDescriptor {
   fn new<T: AsRef<str>>(host: &&(T, Option<u16>)) -> Self {
     NetDescriptor(host.0.as_ref().to_string(), host.1)
   }
 
   pub fn from_string(host: String) -> Self {
+    if is_cidr(&host) {
+      return NetDescriptor(host, None);
```

Comment:
So no port support?

Reply:
Yes, because It becomes hard with IPv6

Reply:
IPv6 addresses should always be contained in square brackets (`[ffff::]`). It should be unambiguous:

`127.0.0.1/24:433`
`[ffff:]/48:433`

Reply:
We _can_ enforce such for Deno, but it's not like so by default in [IPv6 CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation)